src/app/vendors/vendors.scss

```diff
@@ -39,14 +39,11 @@
   }
 
   .xyz-active-throb {
-    width: 30px;
-    height: 30px;
     border-radius: 100px;
-    margin-right: 20px;
```

Comment:
Without the margin on the right, the active throbber was obscuring vendor information.

Was it still legible with it removed?

Reply:
I switched from having two classes (glowing and not glowing) that duplicated the basic styles, to having one style that applied the base non-glowing style and one that adds glowing. The base style is always present, and now we just switch on and off the glowing class.

Reply:
Ah, you're right - the pull elided the base style with the margin on it.